### ISSUE TITLE
Please add "Return Err" in sendmails.rs

### ISSUE THREAD

**Issue by suzhouclark** [AarambhDevHub/rust-backend-axum#2]
Please add "Return Err" in sendmails.rs.
Otherwise, user will not get info, even there is some error when sending mails.

**Comment by aarambh-darshan** [comment]
You're absolutely right — I missed that part. Without returning `Err`, the user won't be informed about email send failures. Thanks for pointing it out! I'll update `sendmails.rs` and add proper error handling to return the error when sending mails fails.

Appreciate the helpful feedback!
